pkg/solver/server.go

```diff
@@ -78,6 +78,9 @@ func (solverServer *solverServer) ListenAndServe(ctx context.Context, cm *system
 	subrouter.HandleFunc("/job_offers", http.GetHandler(solverServer.getJobOffers)).Methods("GET")
 	subrouter.HandleFunc("/job_offers", http.PostHandler(solverServer.addJobOffer)).Methods("POST")
 
+	subrouter.HandleFunc("/job_offers/{id}", http.GetHandler(solverServer.getJobOffer)).Methods("GET")
+	subrouter.HandleFunc("/job_offers/{id}/results", solverServer.getJobOfferResult).Methods("GET")
```

Comment:
What would you think about renaming this endpoint to `/job_offers/{id}/files`?

The other route that is used for downloading job outputs is `/deals/{id}/files`, so `files` would be consistent with that one. "results" is overloaded because we use it for retrieving results metadata in `/deals/{id}/result`.